src/rpdk/core/project.py

```diff
@@ -421,14 +421,36 @@ def load_hook_schema(self):
         with self.schema_path.open("r", encoding="utf-8") as f:
             self.schema = load_hook_spec(f)
 
-    def load_schema(self):
+    def load_schema(self, args=None):
         if not self.type_info:
             msg = "Internal error (Must load settings first)"
             LOG.critical(msg)
             raise InternalError(msg)
 
+        type_name = f"{self.type_info[0]}::{self.type_info[1]}::{self.type_info[2]}"
+
+        original_schema = self._retrieve_global_schema(type_name, args)
```

Comment:
Is it global?  Isn't this just the regional schema for the same resource type?

Reply:
yea it's regional